deploy/openshift/crds.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
deploy/kubernetes/crds.yaml is missing ? or we don't plan to add runtimes outside openshift ?

Reply:
you're right I forgot it

Reply:
@lucasponce added